course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -99,14 +146,129 @@
 
                 courseInstance['sites'] = ci.sites;
             }
-            courseInstance['members'] = members.count;
 
             return courseInstance;
         };
 
-        dc.courseInstanceId = $routeParams.courseInstanceId;
+        dc.resetForm = function() {
+            dc.formDisplayData = angular.copy(dc.courseInstance);
+        };
 
-        dc.setCourseInstance($routeParams.courseInstanceId);
+        dc.resetFormFromUI = function() {
+            dc.resetForm();
+            dc.resetGlobalAlerts();
+            dc.alerts.global.formReset = {show:true};
+        };
 
+        dc.resetGlobalAlerts = function(scrollToAlerts) {
```

Comment:
I know `resetGlobalAlertsAndMaybeScrollToTop()` is awkward, but I wouldn't expect something called `resetGlobalAlerts()` to change the viewport at all.  Can we rename or split it out?